packages/components/src/Button/__snapshots__/button.spec.tsx.snap

```diff
@@ -103,10 +103,21 @@ exports[`<Button /> outlineWithIcon story renders snapshot 1`] = `
 exports[`<Button /> withDataTestId story renders snapshot 1`] = `
 <button
   class="button sizeMedium variantFlat colorAlert"
-  data-test-id="fake-test-id"
```

Comment:
I always make this typo.

Reply:
We could use a lint rule for this. Helps align puppeteer and testing-library usage, as well (since puppeteer or the like don't really care what data-* attributes we use, but testing-library does).

Reply:
Do you think it's common enough to make that useful? I only know that I'm constantly mixing it up, but I assumed it was just me.

Reply:
I think it's probably useful because of how easy the rule is to write (detect jsx attributes of `data-test-id`).

But the hard part may be figuring out where to put it 🤔 . Directly into this repo? In a new eslint plugin in https://github.com/chanzuckerberg/frontend-libs ?

Reply:
Probably what we want is a new eslint-plugin-testing package in frontend-libs that contains a data-testid rule and Devin's [no-wrapper-snapshot](https://github.com/chanzuckerberg/along/blob/231e447b96b7e7839a0e178db351162138e03a4c/client/src/eslint/no-wrapper-snapshot.js) rule from Along.

Reply:
^testing package would be cool!